docs/DeveloperGuide.md

```diff
@@ -326,18 +326,48 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 *{More to be added}*
 
+
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+**Technical requirements:**
+
+1.  The product should work on any _mainstream OS_ as long as it has Java `11` installed.
+
+**Quality requirements:**
+
+1.  A user with above average typing speed for plain English text should be able to accomplish most of the tasks using commands faster than with the mouse.
+
+**Constraints:**
```

Comment:
I feel that constraints is vague. How about maintability requirement because testability is mentioned.

Reply:
can reference here for others: https://www.guru99.com/non-functional-requirement-type-example.html

Reply:
I got `Constraints` from the concrete NFR examples given in the textbook: https://nus-cs2103-ay2122s2.github.io/website/se-book-adapted/chapters/requirements.html#non-functional-requirements

Another alternative we can have is `Testability`?

Lemme know your thoughts

Reply:
Yup, I think testability will work. I thnk constraints can be used if it is hard to pinpoints exactly the nature of the requirements but here testability is mentioned so I think better go with that.